sklearn/tests/test_calibration.py

```diff
@@ -376,6 +377,31 @@ def decision_function(self, X):
     calibrated_clf.fit(X, y)
 
 
+def test_isotonic_calibration_strict():
+    """
```

Comment:
we use comments instead of docstrings for tests

Reply:
Ok will change that ! I just did as [the test above](https://github.com/scikit-learn/scikit-learn/blob/6847c22b6fd9cb9e901eec66884d2d011f24a628/sklearn/tests/test_calibration.py#L361). 
Perhaps, we'll save another PR to change all of the docstrings for tests in here ?